src/main/java/org/jenkinsci/plugins/orgfolder/github/MainLogic.java

```diff
@@ -0,0 +1,204 @@
+                if (item.getView("Branches")==null && item.getView("All") instanceof AllView) {
+                    // create initial views
+                    ListView bv = new ListView("Branches");
+                    DescribableList<ListViewColumn, Descriptor<ListViewColumn>> cols = bv.getColumns();
+                    JobColumn name = cols.get(JobColumn.class);
+                    if (name!=null)
+                        cols.replace(name,new CustomNameJobColumn(Messages.class, Messages._ListViewColumn_Branch()));
+                    bv.getJobFilters().add(new BranchJobFilter());
+
+                    ListView pv = new ListView("Pull Requests");
+                    cols = pv.getColumns();
+                    name = cols.get(JobColumn.class);
+                    if (name!=null)
+                        cols.replace(name,new CustomNameJobColumn(Messages.class, Messages._ListViewColumn_PullRequest()));
+                    bv.getJobFilters().add(new PullRequestJobFilter());
+
+                    item.addView(bv);
+                    item.addView(pv);
+                    item.deleteView(item.getView("All"));
+                    item.setPrimaryView(bv);
+                }
+
+                bc.commit();
+            } finally {
+                bc.abort();
+                UPDATING.get().remove(item);
+            }
+        }
+    }
+
+    /**
+     * Applies UI customizations to two levels below {@link OrganizationFolder}, which maps to a branch.
+     */
+    public void applyBranch(WorkflowJob branch, WorkflowMultiBranchProject repo, GitHubSCMNavigator scm) throws IOException {
+        if (UPDATING.get().add(branch)) {
+            Branch b = repo.getProjectFactory().getBranch(branch);
+            GitHubLink repoLink = repo.getAction(GitHubLink.class);
+            if (repoLink!=null) {
+                BulkChange bc = new BulkChange(branch);
+                try {
+                    branch.replaceAction(new GitHubLink("branch",repoLink.getUrl()+"/tree/"+b.getName()));
+                    bc.commit();
+                } finally {
+                    bc.abort();
+                    UPDATING.get().remove(branch);
+                }
+            }
+        }
+    }
+
+    GitHub connect(SCMSourceOwner of, GitHubSCMNavigator n) throws IOException {
+        StandardCredentials credentials = Connector.lookupScanCredentials(of, n.getApiUri(), n.getScanCredentialsId());
+        return Connector.connect(n.getApiUri(), credentials);
+    }
+
+    /**
+     * Verify if exists a webhook by its URL.
+     */
+    private boolean existsHook(GHOrganization org, String url) throws IOException {
+        for (GHHook hook : org.getHooks()) {
+            if (hook.getConfig().get("url").equals(url)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static MainLogic get() {
+        return Jenkins.getActiveInstance().getInjector().getInstance(MainLogic.class);
```

Comment:
That'd be going backward. This is the fallback method, and injection should be preferred.